lib/chartmogul/resource_path.rb

```diff
@@ -43,5 +46,12 @@ def apply_named_params(params)
         end
       end
     end
+
+    def validate_params!(params)
+      invalid = params.keys.find { |param| INVALID_PARAMS.include?(param) }
```

Comment:
Might keys contain a string?

Reply:
Faraday treats parameters as symbols as far as I can tell, which is also why we do some wrangling of the params here: https://github.com/chartmogul/chartmogul-ruby/blob/c8aa9f89401203246bfc3559738e4bf46669c6b7/lib/chartmogul/resource_path.rb#L14-L16